cg/meta/deliver/utils.py

```diff
@@ -0,0 +1,127 @@
+import logging
+import os
+from pathlib import Path
+from cg.constants import delivery as constants
+from cg.constants.constants import Workflow
+from cg.store.models import Case, Sample
+from housekeeper.store.models import File
+
+LOG = logging.getLogger(__name__)
+
+
+def get_delivery_scope(delivery_arguments: set[str]) -> tuple[bool, bool]:
+    """Returns the scope of the delivery, ie whether sample and/or case files were delivered."""
+    case_delivery: bool = False
+    sample_delivery: bool = False
+    for delivery in delivery_arguments:
+        sample_delivery: bool = is_sample_delivery(delivery)
+        case_delivery: bool = is_case_delivery(delivery)
+    return sample_delivery, case_delivery
```

Comment:
Is it not really weird that we iterate over all "delivery arguments", but we set the scope to that of the last delivery argument? Does this mean that we actually only ever provide a single delivery argument?

Reply:
"Delivery argument" means workflow (usually mip-dna + fastq) or something. Can be removed once we stop passing the workflow as an argument and retrieve it from the case instead